spec/acceptance/multiple_definition_functions_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "acceptance_helper"
+
+describe "Multiple definition functions" do
```

Comment:
What do you think of merging this test into `spec/acceptance/user_manages_functions_spec.rb` instead?